tests/http/server.cpp

```diff
@@ -9,34 +9,79 @@
 #include <zeroeq/http/server.h>
 #include <zeroeq/subscriber.h>
 #include <zeroeq/uri.h>
-#include <zmq.h>
 #include <servus/serializable.h>
 #include <servus/uri.h>
 #include <boost/test/unit_test.hpp>
 #include <thread>
 
+#include <boost/network/protocol/http/client.hpp>
+#include <boost/network/protocol/http/server.hpp>
+
 namespace
 {
+namespace http = boost::network::http;
+// default client from cppnetlib is asynchronous, not handy for tests
+using HTTPClient =
+    http::basic_client< http::tags::http_default_8bit_tcp_resolve, 1, 1 >;
+
+using ServerReponse = http::basic_response<http::tags::http_server>;
+HTTPClient::response _buildResponse( const std::string& body = std::string( ))
+{
+    HTTPClient::response response;
```

Comment:
It seems that you have Cors Headers implemented several times with the same values (_buildResponse, _buildError and _addCorsHeaders). Any chance this could be put in a single place?